core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DriverComponent.scala

```diff
@@ -38,10 +38,6 @@ trait DriverComponent extends StringValidationUtils {
     }
   }
 
-  def createBatches[T](items: Set[T], batchSize: Int = 1000): Iterable[Set[T]] = {
-    items.zipWithIndex.groupBy(_._2 % batchSize).values.map(_.map(_._1))
-  }
-
```

Comment:
This function should use int division rather than modulo to achieve its intent, but `grouped` does the intended thing without writing any code so I'm replaced invocations of this function with `grouped`